examples/08 ovftool on ESXi/README.md

```diff
@@ -0,0 +1,32 @@
+# Terraform esxi Provider (08 ovftool on ESXi)
+---
+
+Since we have SSH enabled on ESXi, it's possible to install ovftool directly on the ESXi host and avoid large network copies
+by hosting the OVA/OVFs locally. 
+
+To use this functionality, perform the following steps:
```

Comment:
How to Do that man , there are only win and linux files meanwhile esxi is not linux , tho you do it with host client ? i dont found any console or sth to some cmds , am a beginner..